Adds html pages and content based on wireframe sketches

krichmond06 opened the pull request.

krichmond06 committed:
Adds html pages and content based on wireframe sketches

axamei commented on contact.html:
```diff
@@ -0,0 +1,37 @@
+  <meta charset="utf-8">
+  <meta name="Description" content="INF 7420 Project">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Document</title>
```
I recommend changing the title of your pages.

axamei commented on contact.html:
```diff
@@ -0,0 +1,37 @@
+    <img src="images/selfpohtoflyfish.jpg" alt="photo of katie richmond dressed in fly fishing gear and standing on a paved path with trees in background"/>
+       <p>Adventures abound wherever you are!</p>
+       <p>Thanks for visiting my page!</p>
+  <div id="footer">
```
Instead of the div, I recommend using the `<footer>` element.

axamei reviewed (changes requested):
In addition to the footer, I recommend adding a `<header>` and `<main>` element with the appropriate elements nested inside those tags. This will create a more meaningful structure for your pages.

krichmond06 committed:
Adds header, main, and footer elements and changes title as suggested by Axa

krichmond06 commented:
Thank you for your suggestions, Axa! I have made the suggested changes and pushed the new commit. Thanks!

axamei reviewed (changes requested):
I recommend you change the title of your pages / the text between the `<title></title>` tags in the head of the document.

krichmond06 commented:
Oops, I changed the title on the index page, but forgot to change it on the others. I have changed it now, but am running into an issue with pushing the changes. I keep getting this message when trying to add files to commit: "Another git process seems to be running in this repository, e.g.
an editor opened by 'git commit'. Please make sure all processes
are terminated then try again. If it still fails, a git process
may have crashed in this repository earlier:
remove the file manually to continue." I've closed all other processes, so I'm not sure what the issue is?

krichmond06 committed:
Adds title to contact and my-work page

krichmond06 commented:
I believe I deleted the correct file that was causing the issue and have pushed the changes now.
